package.json

```diff
@@ -16,59 +16,52 @@
   ],
   "main": "highcharts/module.js",
   "scripts": {
-    "dev": "nuxt",
-    "build": "nuxt build",
-    "start": "cross-env node server/index.js",
-    "generate": "nuxt generate",
-    "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
+    "dev": "nuxt-ts",
+    "build": "nuxt-ts build",
+    "start": "nuxt-ts start",
+    "generate": "nuxt-ts generate",
+    "lint:js": "eslint --ext .js,.vue --ignore-path .gitignore .",
+    "lint": "npm run lint:js",
     "test": "cross-env NODE_ENV=test npm run test:specs:cov && npm run test:e2e:cov",
-    "test:specs": "cross-env TEST=specs ava --config specs.config.js",
+    "test:unit": "cross-env TEST=unit ava ./test/specs/**/*",
     "test:specs:cov": "cross-env TEST=specs nyc ava --timeout=10m --config specs.config.js",
     "test:specs:watch": "cross-env TEST=specs ava --config specs.config.js --color --watch",
-    "test:e2e": "cross-env TEST=e2e ava --config e2e.config.js",
+    "test:e2e": "cross-env TEST=e2e ava ./test/e2e/**/*",
     "test:e2e:cov": "cross-env TEST=e2e nyc --no-clean ava --timeout=10m --config e2e.config.js",
     "test:e2e:watch": "cross-env TEST=e2e ava --config e2e.config.js --color --watch"
   },
   "dependencies": {
-    "highcharts": "^8.1.0"
+    "@nuxt/typescript-runtime": "^2.0.0",
```

Comment:
Does this alone fix the "require.context" issue when someone installs the library? Or will they also need @nuxt/typescript-build?

Reply:
even now using the Nuxt TS would actually add `@nuxt/typescript-build` to the deps and in the 'how to' it is mentioned as well.
so if users want the TS they do need the package (until v3 is released and we will need to figure out what and how)